include/swift/AST/DiagnosticEngine.h

```diff
@@ -434,6 +434,7 @@ namespace swift {
     enum class Behavior : uint8_t {
       Unspecified,
       Ignore,
+      Remark,
```

Comment:
Nitpick: Remark should probably come after Note, so that the standalone diagnostics are all contiguous.